.github/workflows/quick-test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         driver: [bazel, super-test]
         include:
           - driver: bazel
-            run-test: cd c++ && bazel test --verbose_failures --test_output=errors -k //...
+            run-test: cd c++ && bazel test --verbose_failures --test_output=errors -k --cxxopt='-Werror' //...
```

Comment:
I was concerned about Werror slowing down local development but this way it is only applied on CI, that works well for me.